civcraftConfigs/tetconfig.yml

```diff
@@ -163,4 +261,66 @@ monster:
       buffs:
             Renenseanse:
                        type: RESISTANCE
-                       level: 1      
\ No newline at end of file
+                       level: 1      
+    firespider:
+      type: SPIDER
+      name: Fire Spider
+      spawn_chance: 0.4
+      identifier: firespider
+      on_hit_message: STOP BOTTING AND FITE M8
+      on_hit_debuffs:
+            wither:
+                       type: WITHER
+                       level: 1
+                       duration: 3s
+                       chance: 0.5 
+spawner:
```

Comment:
oh i see i think aki was tryin to make mob spawn in a spcific area we didnt realize that option was only for spawners,